apps/zotonic_filehandler/src/zotonic_filehandler.erl

```diff
@@ -17,22 +17,14 @@
 %% See the License for the specific language governing permissions and
 %% limitations under the License.
 
-
 -module(zotonic_filehandler).
 
--export([
-    compile_all/0,
-    compile_all_sync/0,
-    reload_modules/0,
-    reload_module/1,
-    load_module/1,
-    compile_options/1,
-    terminal_notifier/1
-]).
+-export([compile_all/0, compile_all_sync/0, reload_modules/0, reload_module/1,
+         load_module/1, compile_options/1, terminal_notifier/1]).
```

Comment:
Personally, I find it easier to see what is exported by having all functions on their own line.